builder.go

```diff
@@ -94,7 +94,7 @@ func (b Builder) Build(ctx context.Context, outputFile string) error {
 	log.Println("[INFO] Building k6")
 
 	// tidy the module to ensure go.mod and go.sum are consistent with the module prereq
-	tidyCmd := buildEnv.newCommand("go", "mod", "tidy")
+	tidyCmd := buildEnv.newCommand("go", "mod", "tidy", "-compat=1.17")
```

Comment:
Wouldn't this force 1.17 as the minimum supported version for building? Do we want to impose that requirement? :thinking: